release/helm/Dockerfile

```diff
@@ -16,10 +16,15 @@ ENV OPERATOR=/usr/local/bin/helm-operator \
 
 # install operator binary
 COPY --from=builder /go/src/github.com/operator-framework/operator-sdk/build/operator-sdk-dev ${OPERATOR}
-COPY bin /usr/local/bin
+COPY release/helm/bin /usr/local/bin
 
 RUN /usr/local/bin/user_setup
 
+# Ensure directory permissions are properly set
```

Comment:
Do we need to fix this in the scaffolding upstream? This should probably be a part of the user_setup script if it's necessary for the helm operator to run

Reply:
I believe that it is not required too. I just added to keep equals ansible which is working in the OCP CI and to build the downstream mage. I will remove it.